components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/Utils.java

```diff
@@ -57,4 +59,11 @@
         }
         return null;
     }
+
+    public static void setJsonPayLoad(org.apache.axis2.context.MessageContext axis2MessageContext, JSONObject payload){
+        JsonUtil.newJsonPayload(axis2MessageContext, payload.toString(),  true, true);
```

Comment:
Why do we use a deprecated method here? Aren't there any alternative?

Reply:
Fixed in 5e9e894